vmdb/lib/extensions/as_const_missing.rb

```diff
@@ -0,0 +1,74 @@
+module ActiveSupport
+  module Dependencies
+    module ModuleConstMissing
```

Comment:
Let's list where this came from:  https://github.com/rails/rails/blob/3-2-stable/activesupport/lib/active_support/dependencies.rb#L155

Also, we should say we're reopening the module to wrap the `const_missing` call